packages/editor/src/pages/Editor/components/Main/Header/index.tsx

```diff
@@ -56,8 +57,8 @@ interface IProps {
   dispatch: Dispatch;
 
   activeSolution: ISolution;
-  notifyClipboardCopySuccess();
-  notifyClipboardCopyFailure();
+  notifyClipboardCopySuccess: () => void;
```

Comment:
out of curiosity, what complained that made you switch to `() => void`?

Reply:
tslinter thought that the return type was any.

Reply:
ah gotcha